modules/theme/src/themes/default/assets/images/icons/plus-icon.svg

```diff
@@ -0,0 +1,25 @@
+<!--
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg version="1.1" id="plus-icon" class="icon" xmlns="http://www.w3.org/2000/svg" width="14" height="14" x="0px" y="0px" viewBox="0 0 512 512" xml:space="preserve">
+	<g>
+		<g>
```

Comment:
Please check if the file has been formatted correctly.

Reply:
This might be due to the SVG rendering in Github. The formatting looks fine in the IDE.